codes/10775_py3_sjskgks.py

```diff
@@ -0,0 +1,22 @@
+import sys
+input = sys.stdin.readline
+
+numOfGate = int(input())
+numOfPlane = int(input())
+possibleDock = [i for i in range(numOfGate+1)]
+
+def find_gate(try_dock):
+    global possibleDock
+    if possibleDock[try_dock]==try_dock:
+        possibleDock[try_dock]=possibleDock[try_dock-1]
+        return try_dock

```

Comment:
이곳에서 possibleDock[try_dock]으로 바꾸면 나중에 서치할때 하나 덜 보게 되어 조금 더 빠릅니다.
```suggestion
        return possibleDock[try_dock]
```